### ISSUE TITLE
Warnings while compiling

### ISSUE THREAD

**Issue by viegelinsch** [haught/cws_plugin#1]
Hi,
I got some warnings while compiling:

cws_plugin.cpp:156:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%04d.png";

cws_plugin.cpp:157:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%04d.png";

cws_plugin.cpp:160:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%05d.png";

cws_plugin.cpp:161:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%05d.png";

cws_plugin.cpp:164:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%06d.png";

cws_plugin.cpp:165:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%06d.png";

cws_plugin.cpp:168:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%07d.png";

cws_plugin.cpp:169:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%07d.png";

cws_plugin.cpp:172:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%08d.png";

cws_plugin.cpp:173:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%08d.png";

cws_plugin.cpp:176:12: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                format = "%s/1%020d.png";

cws_plugin.cpp:177:16: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
                fileFormat = "1%020d.png";

Maybe you want to look into it?!
If not, just close this issue.
Unfortunately C++ ist not my strong side. 😇

**Comment by benmcl** [comment]
Yep, same warnings as viegelinsch above, along with his caveat "C++ is not my strong side". I'm using macOS 10.15.5 ... updated everything....

Even with the errors, the cws plugin is still created. I went ahead and installed it into Chitubox, then sliced a simple cylinder and exported as a .cws file for the machine. When I looked at the cws file, there was a gcode file and a png for every layer: 10001.png, 10002.png, ... But it looks like the gcode file only moves the toolhead up/down and turns the fan on/off, but fails to load the pngs. Below is a snippet of gcode from one layer ... the other layers are similar. I did not try it on the printer because I didn't see the 3d print happening. Thanks for your efforts and sharing your code!!

```
;********** Pre - Slice Start ********
;Set up any GCode here to be executed before a lift
;********** Pre - Slice End **********
;<Slice> 3
;<Delay> 50000
;<Slice> Blank
M106 S0
; ********** Lift Sequence ********
G1 Z5 F65
G1 Z-4.95 F150
;<Delay> 0
M106 S255
```
